application/values.yaml

```diff
@@ -148,54 +177,105 @@ service:
   #   protocol: TCP
   #   targetPort: 8081
 
+# Ingress object for exposing services
 ingress:
   enabled: false
+  
+  # Port of the service that serves pods
   servicePort: 8080
+  
+  # List of host addresses to be exposed by this Ingress
   hosts:
     - chart-example.local
-  annotations: 
+  
+  # Add annotations to this Ingress
+  annotations:
     # kubernetes.io/ingress.class: external-ingress
     # ingress.kubernetes.io/rewrite-target: /
     # ingress.kubernetes.io/force-ssl-redirect: true
+
+  # TLS details for this Ingress
   tls:
     # Secrets must be manually created in the namespace.
     # - secretName: chart-example-tls
     #   hosts:
     #     - chart-example.local
 
-routes:
+# Route object for exposing services (OpenShift only)
+route:
   enabled: false
-  annotations: {}
+
+  # Add annotations to this Route
+  annotations:
     # kubernetes.io/ingress.class: external-ingress
     # ingress.kubernetes.io/rewrite-target: /
     # ingress.kubernetes.io/force-ssl-redirect: true
-  additionalLabels: {}
+  
+  # Additional labels for this Route
+  additionalLabels:
+
+  # Host addresses to be exposed by this Route
   host: chart-example.local
```

Comment:
I think this should be left empty by default, as then Openshift automatically configures host, i.e. <route-name>-<namespace>.<domain>